homeassistant/components/tradfri/sensor.py

```diff
@@ -64,8 +65,12 @@ def _get_air_quality(device: Device) -> int | None:
 
 def _get_filter_time_left(device: Device) -> int:
     """Fetch the filter's remaining life (in hours)."""
+    assert device.air_purifier_control is not None
     return round(
-        device.air_purifier_control.air_purifiers[0].filter_lifetime_remaining / 60
+        cast(
+            int, device.air_purifier_control.air_purifiers[0].filter_lifetime_remaining
```

Comment:
Could `device.air_purifier_control.air_purifiers[0].filter_lifetime_remaining` be lifted into a local? Would make for an easier-to-read expression on this line...

Reply:
That wouldn't change much unfortunately. The name is just to long.

I would recommend working on the update for `pytradfri` to either version `10.x` or `11.x`. Both include type hints which will make the `cast` here unnecessary.

From what I can tell, it's mostly type hints which have been added / improved, so an update might be fairly strait forward. However, I don't have any way to test it, so it's better if someone else would do it.